stl/inc/ranges

```diff
@@ -6681,14 +6681,8 @@ namespace ranges {
                             "cannot advance chunk_view iterator past end");
                     }
 #endif // _ITERATOR_DEBUG_LEVEL != 0
-
-                    // Per to-be-filed LWG issue (See GH-2995)
-                    if constexpr (sized_sentinel_for<_Base_sentinel, _Base_iterator>) {
-                        _Missing = _RANGES advance(_Current, _Count * _Off, _End);
-                    } else {
-                        _Current += static_cast<difference_type>(_Count * (_Off - 1));
-                        _Missing = _RANGES advance(_Current, _Count, _End);
-                    }
+                    _Current += static_cast<difference_type>(_Count * (_Off - 1));
+                    _Missing = _RANGES advance(_Current, _Count, _End);

```

Comment:
This changes removes the optimization we were performing for the `sized_sentinel_case`. Is there any particular reason to do so, or should we restore the code and simply remove the "LWG issue" comment? (Comment applies to both places.)